roles/users/tasks/update_keys.yml

```diff
@@ -1,9 +1,43 @@
 ---
-- name: Update authorized_keys for each user.
+- name: Merge managed_users and managed_admin users
+  set_fact:
+    pubkey_users: "{{ managed_users|list + managed_admin_users|list }}"
+
+# The following set_fact calls are apparently necessary to avoid using sudo on
+# localhost to clone the keys repo.
+- set_fact:
+    ansible_become_orig: "{{ ansible_become }}"
+
+- set_fact:
+    ansible_become: false
+
+- name: Clone the keys repo
```

Comment:
also, should we add accept_hostkey because of this:  http://docs.ansible.com/ansible/git_module.html#notes ?